src/helpers/user.helper.js

```diff
@@ -27,14 +28,24 @@ export const getUserIndexByAccountNumber = (accountNumber) => {
   return userIndex;
 };
 
-export const getUserTransactions = (accountNumber = getLoggedInUserAccountNumber()) => {
+export const getUserTransactions = (
+  accountNumber = getLoggedInUserAccountNumber()
+) => {
   const user = getUserByAccountNumber(accountNumber);
   return user?.transactions || [];
-}
+};
+export const getUserAccountName = (
```

Comment:
This function adds unnecessary granularity to your code. You can get any user data from the user object returned by `getUserByAccountNumber()`.

Reply:
Done